test-app/src/widgets/Button.js

```diff
@@ -4,19 +4,10 @@ import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
 export default class Button extends createjs.Container {
   constructor(options, tabIndex, callBack = _.noop) {
     super();
-    _.bindAll(
-      this,
-      '_onFocus',
-      '_onBlur',
-      '_onMouseDown',
-      '_onMouseUp',
-      '_onClick'
-    );
+    _.bindAll(this, '_onFocus', '_onBlur', '_onClick');
     this.callBack = callBack;
     this.addEventListener('focus', this._onFocus);
     this.addEventListener('blur', this._onBlur);
-    this.addEventListener('mousedown', this._onMouseDown);
-    this.addEventListener('mouseup', this._onMouseUp);
```

Comment:
- can you tell me why these listeners were removed?

Reply:
I removed these handlers who were responsible for hiding the background including the borders which looked like something shouldn't happen when interacted
https://github.com/CurriculumAssociates/createjs-accessibility/pull/94/files#diff-9b7586c142515f4edc22cfd6efdfeead4fc4240f14f1a06d08b36e31b99eebdcL94-L101